### PR TITLE
Update Palaeontology and Journal of Paleontology

### PR DESCRIPTION
**Pull request by ms609**
These updates reflect edits that I have been asked to make by the journals' copyeditors, and bring the styles up to date.

### CODE REVIEW

File: journal-of-paleontology.csl [hunk 1]
```diff
@@ -179,7 +179,7 @@
                 <choose>
                   <if is-numeric="volume">
                     <group delimiter=" ">
-                      <text value="Vol. "/>
```
Comment (adam3smith) [inline journal-of-paleontology.csl]: I'd much rather do this properly with label so here `<label variable="volume" form="short"/>` and then define
```
<locale>
  <terms>
    <term name="volume" form="short">v.</term>
  </terms>
</locale>
```

after line 20. above

File: journal-of-paleontology.csl [hunk 2]
```diff
@@ -152,10 +152,10 @@
                 <text variable="page"/>
               </else>
             </choose>
-            <text term="page" form="short"/>
+            <text term="page" form="short" prefix="p. "/>
```
Comment (adam3smith) [inline journal-of-paleontology.csl]: this isn't right. The `term="page"` already gives you `p.` or `pp.` so you definitely don't want a `p. ` prefix on it. What's the intended end-result here?

File: journal-of-paleontology.csl [hunk 3]
```diff
@@ -205,11 +205,11 @@
...
           </else-if>
           <else>
-            <text variable="title" suffix=". "/>
+            <text variable="title" suffix=": "/>
             <text variable="container-title" form="long" text-case="title" suffix=","/>
-            <group delimiter=":">
-              <text variable="volume" prefix=" "/>
-              <text variable="page"/>
+            <group delimiter=", ">
+              <text variable="volume" prefix="v. "/>
```
Comment (adam3smith) [inline journal-of-paleontology.csl]: same here -- better to put the v. and the p. into labels. For the p.: this is never going to be pp.?

File: palaeontology.csl [hunk 4]
```diff
@@ -142,14 +144,14 @@
...
+              <text term="in" text-case="capitalize-first" font-style="italic" suffix=" "/>
               <text macro="editor" suffix=" "/>
               <group>
                 <text variable="container-title" text-case="title" font-style="italic" suffix=", "/>
                 <choose>
                   <if is-numeric="volume">
-                    <group delimiter=" ">
-                      <text value="Vol. "/>
+                    <group delimiter=" " suffix=".">
+                      <text value="Vol."/>
```
Comment (adam3smith) [inline palaeontology.csl]: let's use term here as you do in line 118 above

File: journal-of-paleontology.csl (hunk 2 above)
Comment (ms609) [reply journal-of-paleontology.csl]: This was an error. Thanks for spotting it.

File: journal-of-paleontology.csl (hunk 3 above)
Comment (ms609) [reply journal-of-paleontology.csl]: Apparently p. is to be used whether referring to one page or to a range.

File: journal-of-paleontology.csl (hunk 1 above)
Comment (adam3smith) [reply journal-of-paleontology.csl]: It's `form="short"`; you have `format="short"` which doesn't exist, so CSL reverts to the default, which is long.

Comment (ms609) [reply journal-of-paleontology.csl]: Whoops! Thanks.  Looking good now.

**Comment by ms609** [comment]
I think this should now be good to go -- did I miss anything, @adam3smith ?

**Comment by adam3smith** [comment]
Great, thanks!

**Pull request merged**